pkg/volume/util/util.go

```diff
@@ -235,3 +239,34 @@ func LoadPodFromFile(filePath string) (*v1.Pod, error) {
 	}
 	return pod, nil
 }
+
+func ZonesSetToLabelValue(strSet sets.String) string {
```

Comment:
Where is this called?

Reply:
In `pkg/cloudprovider/providers/gce/gce_disks.go`

Reply:
Nm the file was collapsed